### ISSUE TITLE
Coverage report not found when using the orb and the -f option

### ISSUE THREAD

**Issue by ryanisnan** [codecov/codecov-circleci-orb#1]
Thanks for putting this orb together.

I've recently tried integrating it into a project, and I've noticed that the upload command is not able to find my coverage reports, despite me indicating where they are located. I have experimented with manually re-creating the step in the orb, and it works, so I am uncertain where the issue lies.

## .circleci/config.yml file - broken version
```
  unittest:
    machine:
      enabled: true
      docker_layer_caching: true
    steps:
      - checkout
      - run:
          name: Install Dependencies.
          command: pip install docker-compose
      - run:
          name: Run those tests.
          command: docker-compose -f docker-compose.test.yml up --build --exit-code-from test
      - run:
          name: List that cov file
          command: cat /tmp/auth/.coverage
      - codecov/upload:
          file: /tmp/auth/.coverage
```

Note that's just one job in the file and not the entire file.

## docker-compose.test.yml
```
version: '3'
services:
  database:
    image: "postgres:latest"
    expose:
      - 5435
    ports:
      - 5435:5432
    environment:
      - POSTGRES_DB=auth_test
      - POSTGRES_USER=auth
      - POSTGRES_PASSWORD=s0_s3cr3t
    logging:
      driver: none
  redis:
    image: "redis:latest"
    logging:
      driver: none
  test:
    stdin_open: true
    tty: true
    build: src
    command: ["/wait-for-it/wait-for-it.sh", "-t", "5", "localhost:5435", "--", "coverage", "run", "management/test.py"]
    volumes:
      - .:/wait-for-it
      - ./src:/app
      - /tmp/auth:/tmp
    depends_on:
      - redis
      - database
    environment:
      ... other env variables related to project
      - PYTHONPATH=$$PYTHONPATH:/app
      - COVERAGE_FILE=/tmp/.coverage
```

I am mapping `/tmp/auth` on the host to `/tmp` in the container. In circle I am validating the output of the script by `cat /tmp/auth/.coverage`. Here is sample output:

```
#!/bin/bash -eo pipefail
cat /tmp/auth/.coverage

!coverage.py: This is a private format, don't read it directly!{"lines":{"/app/management/test.py":[1,2,3,4,5,8,9,11,12,14,17,40,61,62,41,43,46,52,53,56,23,24,26,27,30,31,32,37,57,58],"/app/management/migrate.py":[1,2,3,5,6,8,9,11,14,25,15,20,21,22],"/app/models/__init__.py":[2,3,4,5,6,7,8,9,10,11,14,15,16,17,18,19,20,21,22,23],"/app/models/entity.py":[1,2,3,4,5,6,9,13,15,17,18,19,21,22,23,26,27,28,30,32,33,34,35,36],"/app/database.py":[1,2,4,5,6,7,8,9,10],"/app/models/entity_api_key.py":[1,2,3,4,5,6,7,8,9,10,11,14,18,21,23,25,26,27,28,29,31,33,34,35,37,38,39,42,43,44,46,15,48,49,50,51,52,53],"/app/models/entity_password.py":[1,2,3,4,5,6,7,8,9,10,13,17,19,21,22,23,24,27,28,29,31,32,33,36,37,38,40,48,42,43,44,45,50],"/app/models/permission.py":[1,2,3,4,5,6,7,8,9,12,16,18,20,21,22,23,24,26,27,30,31,32,34,35,36,39,40,41,43,53,45,46,47,48,49,50],"/app/models/permission_set.py":[1,2,3,4,5,6,7,8,9,12,15,17,19,20,21,22,23,25,26,29,30,31,33,34,35,38,39,40,42,44,45,46,47,48,49,50],"/app/models/permission_set_permission.py":[1,2,3,4,5,6,7,8,9,12,15,17,19,20,22,23,24,26,27,28,31,32,33,34],"/app/models/entity_permission.py":[1,2,3,4,5,6,7,8,9,12,15,17,19,20,22,23,24,26,27,28,31,32,33,34,36],"/app/models/entity_permission_set.py":[1,2,3,4,5,6,7,8,9,12,15,17,19,20,22,23,24,26,27,28,31,32,33,35],"/app/models/access_token.py":[1,2,3,4,5,6,7,8,11,18,19,20,22,24,25,26],"/app/models/refresh_token.py":[1,2,3,4,5,6,7,8,9,12,16,17,22,23,25,27,28,29,19,20],"/app/models/migrations/__init__.py":[1],"/app/models/migrations/0001_initial.py":[1,2,3,4,5,6,7,8,9,12,13,15,16,18,19,20,21,22,23,24,25,28],"/app/models/migrations/0002_seed_auth_data.py":[1,2,3,4,5,6,7,8,9,10,11,12,13,16,20,22,24,27,28,29,30,34,39,42,43,44,45,46,47,48,49,50,51,52,54,55,56,57],"/app/utils.py":[1,2,3,4,5,6,7,8,9,10,13,17,43,60,93,104,113,120,132,47,49,50,51,52,53,54,57,20,21,22,35,36,37,38,40,99,100,101,108,109,110,139,141,124,126,114,117,127,129,142,143,65,66,67,68,69,70,71,72,76,78,14,79,84,85,86,87,90],"/app/settings.py":[1,4,5,6,9,10,11],"/app/models/migrations/0003_create_access_refresh_token.py":[1,2,3,6,7,9,10,11,12,15],"/app/tests/endpoint_tests/__init__.py":[1],"/app/tests/endpoint_tests/test_entities.py":[1,2,3,4,5,8...
```

## CircleCI output - broken version
```
#!/bin/bash -eo pipefail
bash <(curl -s https://codecov.io/bash) \
  -F  \
  -f /tmp/auth/.coverage \
  -n ${CIRCLE_BUILD_NUM} \
  -t ${CODECOV_TOKEN} \
  -y .codecov.yml \


  _____          _
 / ____|        | |
| |     ___   __| | ___  ___ _____   __
| |    / _ \ / _` |/ _ \/ __/ _ \ \ / /
| |___| (_) | (_| |  __/ (_| (_) \ V /
 \_____\___/ \__,_|\___|\___\___/ \_/
                              Bash-0b37652


==> Circle CI detected.
    project root: .
--> token set from env
    Yaml found at: .circleci/codecov.yml
==> Running gcov in . (disable via -X gcov)
==> Python coveragepy not found
==> Searching for coverage reports in:
    + .
--> No coverage report found.
    Please visit http://docs.codecov.io/docs/supported-languages
```

You can see that `-f` is being passed properly, but no coverage report was found.

Alternatively, when I update my `.circleci/config.yml` file to do the upload step manually, it works.

## .circleci/config.yml - working version
```
unittest:
    machine:
      enabled: true
      docker_layer_caching: true
    steps:
      - checkout
      - run:
          name: Install Dependencies.
          command: pip install docker-compose
      - run:
          name: Run those tests.
          command: docker-compose -f docker-compose.test.yml up --build --exit-code-from test
      - run:
          name: List that cov file
          command: cat /tmp/auth/.coverage
      - run:
          name: Upload Coverage Results
          command: |
            bash <(curl -s https://codecov.io/bash) -t $CODECOV_TOKEN -f /tmp/auth/.coverage
```

## CircleCI output - working version
```
#!/bin/bash -eo pipefail
bash <(curl -s https://codecov.io/bash) -t $CODECOV_TOKEN -f /tmp/auth/.coverage


  _____          _
 / ____|        | |
| |     ___   __| | ___  ___ _____   __
| |    / _ \ / _` |/ _ \/ __/ _ \ \ / /
| |___| (_) | (_| |  __/ (_| (_) \ V /
 \_____\___/ \__,_|\___|\___\___/ \_/
                              Bash-0b37652


==> Circle CI detected.
    project root: .
    Yaml found at: .circleci/codecov.yml
    -> Found 1 reports
==> Detecting git/mercurial file structure
==> Reading reports
    + /tmp/auth/.coverage bytes=10300
==> Appending adjustments
    http://docs.codecov.io/docs/fixing-reports
    -> No adjustments found
==> Gzipping contents
==> Uploading reports
    url: https://codecov.io
    query: branch=ryan%2Fcodecov&commit=<hash_obfuscated>&build=544&build_url=&name=&tag=&slug=sokanu%2Fauth&service=circleci&flags=&pr=&job=0
    -> Pinging Codecov
    -> Uploading
    -> View reports at https://codecov.io/github/sokanu/auth/commit/<hash_obfuscated>
```

Any thoughts?

**Comment by thomasrockhu** [comment]
Hi @ryanisnan, thanks for the ticket!  I'll have the team take a look today and see why it might not be working for you.

**Comment by ryanisnan** [comment]
Thanks @thomasrockhu - It seems very unlikely that the issue is within the orb, after reading through the source, but it is reproducible by replacing the orb.

**Comment by thomasrockhu** [comment]
Hi @ryanisnan , thanks for your patience.  This should now be fixed with the latest release `1.0.2`.  Let me know if it doesn't solve the issue.

**Comment by ryanisnan** [comment]
Thanks @thomasrockhu!
